client/src/components/containers/DriverPickUpForm.tsx

```diff
@@ -17,15 +17,24 @@ const INIT_POS = {
 };
 
 export default function DriverPickUpForm() {
-  const client = useApolloClient();
   const dispatch = useDispatch();
   const [driverPos, setDriverPos] = useState(INIT_POS);
   const [count, setCount] = useState(0);
-  const { originPosition, destPosition }: any = useSelector(selectMapReducer);
+  const { originPosition }: any = useSelector(selectMapReducer);
   const { trip }: any = useSelector(selectTripReducer);
 
+  useQuery(GET_ORIGIN_POSITION_AND_DESTINATION_POSITION, {
+    variables: { id: trip.id },
+    onCompleted: tripInfo => {
+      dispatch(setOriginPosition({ lat: tripInfo.trip.origin.latitude, lng: tripInfo.trip.origin.longitude }));
+      dispatch(setDestPosition({ lat: tripInfo.trip.destination.latitude, lng: tripInfo.trip.destination.longitude }));
```

Comment:
```suggestion
    onCompleted: ({ trip }) => {
      dispatch(setOriginPosition({ lat: trip.origin.latitude, lng: trip.origin.longitude }));
      dispatch(setDestPosition({ lat: trip.destination.latitude, lng: trip.destination.longitude }));
```
객체 안에 있는 trip property만 사용하고 있어서 한번 펼쳐주는 것도 좋을 것 같습니다.

Reply:
너무 좋은 리뷰입니다
 갑사합니닷